### PR TITLE
[loggingexporter] Add exemplars logging

### PR DESCRIPTION
**Pull request by matej-g**
**Description:**
Adds logging of exemplars to `loggingexporter`, which is currently missing.

**Link to tracking Issue:** 7912

**Testing:** Enhanced existing unit tests + manual testing.

**Documentation:** N/A

### CODE REVIEW

**Review by codeboten — APPROVED** [review]
Thanks for the contribution!

File: exporter/loggingexporter/internal/otlptext/databuffer.go [hunk 1]
```diff
@@ -258,6 +264,32 @@ func (b *dataBuffer) logLinks(description string, sl ptrace.SpanLinkSlice) {
...
+
+	for i := 0; i < se.Len(); i++ {
+		e := se.At(i)
+		b.logEntry("Exemplar #%d", i)
+		b.logEntry("     -> Trace ID: %s", e.TraceID())
+		b.logEntry("     -> Span ID: %s", e.SpanID())
+		b.logEntry("     -> Timestamp: %s", e.Timestamp())
+		switch e.ValueType() {
+		case pmetric.ExemplarValueTypeInt:
+			b.logEntry("     -> Value: %d", e.IntValue())
```
Comment (codeboten) [inline exporter/loggingexporter/internal/otlptext/databuffer.go:282]: Can a test be added for int types?

File: exporter/loggingexporter/internal/otlptext/databuffer.go [hunk 2]
```diff
@@ -258,6 +264,32 @@ func (b *dataBuffer) logLinks(description string, sl ptrace.SpanLinkSlice) {
...
+		case pmetric.ExemplarValueTypeDouble:
+			b.logEntry("     -> Value: %f", e.DoubleValue())
+		}
+		if e.FilteredAttributes().Len() == 0 {
```
Comment (codeboten) [inline exporter/loggingexporter/internal/otlptext/databuffer.go:286]: This check is already part of `logAttributes` no?

Comment (matej-g) [reply exporter/loggingexporter/internal/otlptext/databuffer.go:286]: You right. I took this over from other places we're printing out attribute, but the checks are unnecessary. I'll remove them.

**Review by codeboten — APPROVED** [review]
Thanks for addressing my comments @matej-g, change looks great

**Pull request merged**